### ISSUE TITLE
KML Support for GroundOverlay / Rasters

### ISSUE THREAD

**Issue by timwaters** [simopaasisalo/MakeMaps#14]
The KML format is quite extensive and MakeMaps should allow valid GroundOverlay (i.e. raster images) and not just vectors

Example KML files:
http://mapwarper.net/maps/16848.kml 
http://mapwarper.net/maps/16848.kml?DBOX=-110.75095129254342,31.718707654757825,-110.61607597411702,31.865469807778688,2

**Comment by spaasis** [comment]
Hi and thank you for commenting!

MakeMaps is using https://github.com/mapbox/togeojson to handle KML files and their conversion to GeoJSON. As is listed on their readme, it doesn't currently support GroundOverlay, though the reason is not mentioned. I opened an issue https://github.com/mapbox/togeojson/issues/118 to get a comment on it's status, and if there are any plans to implementing support.

**Comment by timwaters** [comment]
:+1:

**Comment by spaasis** [comment]
Okay, like Tom answered in the issue, there is no direct equivalent for GroundOverlay in GeoJSON, so that is not something that they are working on.

There seems to be a plugin which supports GroundOverlay https://github.com/shramov/leaflet-plugins/pull/91. I'll need too look into using this plugin to handle KML loading.

However, if it turns out that GO simply cannot be turned into a GeoJSON format with any plugin, I will have to turn it down. Reasoning behind this is that the whole application basically relies on handling Leaflet L.GeoJSON-objects.

**Comment by timwaters** [comment]
Agreed, so if a Leaflet plugin could be made to work with them, perhaps then we could look forward to it being supported. I think it's only in Google Earth that one can view them nowadays.  Perhaps having some format to load in external baselayers (wms /  tiles) would be nice but I imagine this would be a different feature
